README.md

```diff
@@ -1,55 +1,385 @@
-# `Saturn` — parallelism-safe data structures for Multicore OCaml
+[API reference](https://ocaml-multicore.github.io/saturn/)
 
----
+<!--
+```ocaml
+# #thread
+```
+-->
```

Comment:
This comment can be removed, I guess.

Reply:
It depends.  If you need threads then on some platforms the `#thread` to enable threads has to be first.  Otherwise things don't work.

Reply:
I see. Makes sense to keep it.